### PR TITLE
Add initial code

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

**Comment by jebbinSCL** [comment]
@pallenSL

File: .gitignore [hunk 1]
```diff
@@ -131,9 +131,6 @@ DocProject/Help/*.hhp
 DocProject/Help/Html2
 DocProject/Help/html
 
-# Click-Once directory
-publish/
-
 # Publish Web Output
 *.[Pp]ublish.xml
 *.azurePubxml
```
Comment (pallenSL) [inline .gitignore]: *.secrets is probably a good idea here

Comment (jebbinSCL) [inline .gitignore]: I'm currently ignoring Publish.exe.secrets, which is enough for now. Is it worth doing that? If we do that, then it makes sense to also look out for anything containing the word secret, e.g. secrets.config.

Comment (pallenSL) [inline .gitignore]: I cannot see an ignore for Publish.exe.secrets (?)
*.secrets should be enough - there aren't any secrets.config left (on purpose, as we want app.config checked in)

Comment (jebbinSCL) [inline .gitignore]: It was in the initial commit. You have to expand to see it unfortunately. I meant any file of that form secrets.* or otherwise.

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**